spaceship/lib/spaceship/spaceauth_runner.rb

```diff
@@ -49,6 +49,12 @@ def run
 
       yaml = cookies.to_yaml.gsub("\n", "\\n")
 
+      if mac? && Spaceship::Client::UserInterface.interactive? && agree("🙄 Should fastlane copy the cookie into your clipboard, so you can easily paste it? (y/n)", true)
```

Comment:
Shouldn't this happen after the output of the cookie? I have to be able to look at it first if it makes sense, don't I?

Reply:
I think it makes sense to move it after the output. I will move it.